source/geometry_model/interface.cc

```diff
@@ -122,6 +122,12 @@ namespace aspect
         while ((name.size() > 0) && (name[name.size()-1] == ' '))
           name.erase (name.end()-1);
 
+        // backwards compatibility (rename boundaries to all use "top" and "bottom"
+        if (name == "surface" || name == "outer")
+          name = "top";
+        else if (name == "lower" || name == "inner")
```

Comment:
Does "lower" exist in any model? If this name should be kept, then presumably "upper" should be included in the aliases for "top".